src/som/interpreter/actors/ErrorPromiseNode.java

```diff
@@ -10,21 +10,23 @@
 
 
 @GenerateNodeFactory
-@Primitive(primitive = "actorsError:with:isBPResolver:isBPResolution:")
+@Primitive(primitive = "actorsError:with:entry:isBPResolver:isBPResolution:")
+
```

Comment:
Why the space?
Also, you seem to change the signature of the specialization, I'd assume that primitive signature would need to change then, too?

Reply:
I removed the space.

I am not sure about what you are asking for the signature. The signature of AbstractPromiseResolutionNode was also changed, is that what you mean?